client/resources/kb/_classes/Adress.yclass

```diff
@@ -0,0 +1,19 @@
+interface	Address
```

Comment:
maybe store those files in a place that makes it clear they were poorly formatted on purpose.

Reply:
Why is it poorly formatted, then? xDdd

Reply:
Kinda scary to think that _this_ is representative of real projects xD